src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/cjsm/S3ClientsConfig.kt

```diff
@@ -21,7 +21,6 @@ class S3ClientsConfig(private val s3Config: S3Config) {
   @ConditionalOnExpression("T(org.springframework.util.StringUtils).isEmpty('\${app.s3.localstack-url:}')")
   fun amazonS3(): AmazonS3 =
     AmazonS3ClientBuilder.standard()
```

Comment:
I have a feeling this maybe the only change required according to that thread I mentioned?